src/serializer/serializer.js

```diff
@@ -1842,34 +1841,47 @@ export class Serializer {
   init(filename: string, code: string, map?: string = "",
       sourceMaps?: boolean = false, onError?: (Realm, Value) => void) {
     // Phase 1: Let's interpret.
+    if (this.options.profile) console.time("[Profiling] Interpreting Global Code");
     this.execute(filename, code, map, onError);
```

Comment:
Would be nice if you could extract the "Babel parsing" part of execute. But that can come in a separate pull request.